packages/salesforcedx-visualforce-language-server/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@salesforce/salesforcedx-visualforce-language-server",
+  "description": "Visualforce language server",
+  "version": "40.12.0",
+  "publisher": "salesforce",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.15.0"
+  },
+  "dependencies": {
+    "vscode-css-languageservice": "2.1.9",
+    "vscode-html-languageservice": "2.0.10",
+    "vscode-languageserver": "3.4.2",
+    "vscode-languageserver-protocol": "3.4.2",
+    "vscode-languageserver-types": "3.4.0",
+    "vscode-nls": "2.0.2",
+    "vscode-uri": "1.0.1"
+  },
+  "devDependencies": {
+    "@types/mocha": "^2.2.38",
+    "@types/node": "^6.0.40",
+    "mocha": "3.2.0",
+    "shx": "0.2.2"
+  },
+  "scripts": {
```

Comment:
@DatGuyJonathan - I removed the vscode:publish here so that should prevent it from being published to the marketplace for now. This is the main thing I want you to check (to see if I might have forgotten something).